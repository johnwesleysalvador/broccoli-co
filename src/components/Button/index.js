import React from 'react';
import styled from 'styled-components';
import Button from '@material-ui/core/Button';

export default styled(props => <Button {...props} />)`
    && {
        border-radius: 100px;
    }
`;
