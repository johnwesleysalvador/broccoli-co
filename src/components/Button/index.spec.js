import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Button from '.';

describe('<Button />', () => {
    test('Should Button', () => {
        const render = renderer
            .create(<Button variant="raised" color="primary">
                    Lorem ipsum
            </Button>)
            .toJSON();
        expect(render).toMatchSnapshot();
        expect(render.type).toEqual('button');
    });
});
