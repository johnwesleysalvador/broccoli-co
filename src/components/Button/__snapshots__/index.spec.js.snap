// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Should Button 1`] = `
.c0.c0 {
  border-radius: 100px;
}

<button
  className="MuiButtonBase-root-17 MuiButton-root-1 MuiButton-raised-7 MuiButton-raisedPrimary-8 c0"
  disabled={false}
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-2"
  >
    Lorem ipsum
  </span>
  <span
    className="MuiTouchRipple-root-20"
  />
</button>
`;
